airbyte-server/src/main/java/io/airbyte/server/apis/ConnectorBuilderProjectApiController.java

```diff
@@ -52,6 +55,24 @@ public void deleteProject(final ConnectorBuilderProjectIdWithWorkspaceId connect
     });
   }
 
+  @Override
+  @Post(uri = "/get_with_manifest")
+  @Status(HttpStatus.OK)
+  @Secured({EDITOR})
+  @SecuredWorkspace
+  public ConnectorBuilderProjectRead getProject(final ConnectorBuilderProjectIdWithWorkspaceId connectorBuilderProjectIdWithWorkspaceId) {
+    return ApiHelper.execute(() -> connectorBuilderProjectsHandler.getBuilderProjectWithManifest(connectorBuilderProjectIdWithWorkspaceId));
+  }
+
+  @Override
+  @Post(uri = "/list")
+  @Status(HttpStatus.OK)
+  @Secured({EDITOR})
```

Comment:
what is the editor {EDITOR} role?

Reply:
This is the permission level for cloud - I checked with Jonathan that editor is the right one for us - it's the role a regular user gets when logged in in the frontend for their own workspace. We can expand privileges later on but this is the important one.